pkg/operator/status_test.go

```diff
@@ -621,3 +631,84 @@ func TestInClusterBringUpStayOnErr(t *testing.T) {
 
 	assert.False(t, optr.inClusterBringup)
 }
+
+func TestKubeletSkewSupported(t *testing.T) {
```

Comment:
Right now, we have test for happy path.To make it more robust, it feels like we should add more coverage in this unit tests by adding more test cases like kubelet version is something like  v1.21.0-rc.0+6143dea, when kubelet skew is supported, etc.

Reply:
@sinnykumari Updated the test for both path tests and added more version parsing tests.